rpm2extents.c

```diff
@@ -207,6 +229,22 @@ static rpmRC validator(FD_t fdi, FD_t digesto, FD_t sigo,
     return rc;
 }
 
+static void sanitizeSignatureHeader(Header * sigh)
+{
+    struct rpmtd_s td;
+
+    /* This is inspired by the code in unloadImmutableRegion */
```

Comment:
might be worth linking to this issue: https://github.com/rpm-software-management/rpm/pull/1330

I would assume this is fixing the problem at hand right?

Reply:
I did a small test in a container yesterday:
- I converted the `azure-cli` package with `rpm2extents`
- Then I installed the converted package with `rpm -i` with `reflink` plugin installed

Other than a few warnings ("reflink: falling back to copying bits"), the installation seemed to be successful